packages/app/src/background/services/injector/Injector.ts

```diff
@@ -25,49 +47,72 @@ export class InjectorService {
     return InjectorService.INSTANCE;
   }
 
-  isConnected = async (payload: unknown, { urlOrigin }: IZkMetadata): Promise<unknown> => {
-    await this.injectorHandler.getConnectedIdentityMetadata({}, { urlOrigin });
+  connect = async ({ isChangeIdentity }: IConnectionOptions, { urlOrigin }: IZkMetadata): Promise<void> => {
```

Comment:
Right, but I'll change this API later and will add another method for changing identity. It will have the same connect and disconnect events until we figure out how to change identity in a convenient way for user.